create-todo-app.yaml

```diff
@@ -0,0 +1,58 @@
+- name: Create TODO app
+  hosts: managed-nodes
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: api_key
+      prompt: Trello API key
+
+    - name: trello_token
+      prompt: Trello Token
+
```

Comment:
Board ID is not secret, but in reality you would want it as a var so that it can vary without code changes